web/book/src/tutorial/relations.md

```diff
@@ -1,78 +1,110 @@
 # Relations
 
-PRQL is designed on top of relational algebra, which is the an established data
-model used by modern databases. _Relation_ has a rigid mathematical definition,
-which can be simplified down to "tabular data". See table of `albums` for
-example:
-
-| album_id | title               | artist_id |
-| -------- | ------------------- | --------- |
-| 1        | For Those About ... | 1         |
-| 2        | Balls to the Wall   | 2         |
-| 3        | Restless and Wild   | 2         |
-| 4        | Let There Be Rock   | 1         |
-| 5        | Big Ones            | 3         |
-| 6        | Jagged Little Pill  | 4         |
-
-It is composed of columns, each of which has an unique name and a designated
-data type. `album_id` has a data type of "integer numbers" and `title` has data
-type of "text".
-
-> Side note: most definitions use unordered relations, which cannot have
-> duplicate rows. PRQL defines relations to have an order and to be able to
-> contain duplicate rows.
-
-PRQL is designed to query relations such as the `albums` table above. The most
-basic query would be:
+PRQL is designed on top of _relational algebra_, which is the established data
+model used by modern SQL databases.
+A _relation_ has a rigid mathematical definition,
+which can be simplified to "a table of data".
```

Comment:
Could we reduce the level of technical terms for the tutorial?

I get that we like the term "relation" and everyone else thinks it's "table" — if we do want to say "relation" rather than "table", would just saying "think of a relation as a table" be enough?

Reply:
This needs to go into our Style Guide (or some place where we agree on this stuff.) What do we call these things: Table? Relation? Is there any difference? What do practitioners say?

It'll be easy enough to go through the docs harmonizing the choice once we decide.